modules/fm2.h

```diff
@@ -0,0 +1,61 @@
+#pragma once
+#ifndef DSY_FM2_H
+#define DSY_FM2_H
+
+#include <stdint.h>
+#include "oscillator.h"
+#ifdef __cplusplus
+
+namespace daisysp
+{
+/** Simple 2 operator FM synth voice.
+     
+    Date: November, 2020
+
+    Author: Ben Sergentanis
+*/
+
+class Fm2
+{
+  public:
+    Fm2() {}
+    ~Fm2() {}
+
+    /** Initializes the FM2 module.
+        \param samplerate - The sample rate of the audio engine being run. 
+    */
+    void Init(float samplerate);
+
+
+    /**  Returns the next sample
+    */
+    float Process();
+
+    /** Carrier freq. setter
+	    \param freq Carrier frequency in Hz
+	*/
+    void SetFrequency(float freq);
+
+    /** Set modulator freq. relative to carrier
+		\param ratio New modulator freq = carrier freq. * ratio
+	*/
+    void SetRatio(float ratio);
+
+    /** Index setter
+      \param FM depth, 5 = 2PI rads
+  */
+    void SetIndex(float index);
```

Comment:
Not the biggest deal in the world (and maybe not-noticeable depending on your editor), but the misalignment due to mixed tabs/spaces is pretty hard to look at here. Also happens through out the rest of this file.

This won't fail style check because I don't think it scans within comments, but the fix_style script should still resolve this.